homeworks/14/instagram-api/LogoutServlet.java

```diff
@@ -0,0 +1,29 @@
+package com.example.instagram;
+
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet(name = "LogoutServlet", value = "/logout")
+public class LogoutServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        response.setContentType("text/html");
+        PrintWriter out = response.getWriter();
+        out.println("<form action=\"logout\" method=\"post\" id=\"logout\"></form>");
+        out.println("<input type=\"submit\" form=\"logout\" value=\"Log out\"><br>\n");
+
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        response.setContentType("text/html");
+        PrintWriter out = response.getWriter();

```

Comment:
It is good practice to close resources such as PrintWriters when you are finished using them.